docker-compose.yml

```diff
@@ -1,12 +1,18 @@
 # This configuration file is for the **development** setup.
 # For a production example please refer to getredash/setup repository on GitHub.
-version: '2'
+version: "2"
 x-redash-service: &redash-service
-  build: .
+  build:
+    context: .
+    args:
+      skip_frontend_build: "true"
   volumes:
     - .:/app
 x-redash-environment: &redash-environment
+  REDASH_LOG_LEVEL: "INFO"
   REDASH_REDIS_URL: "redis://redis:6379/0"
+  REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
+  REDASH_RATELIMIT_ENABLED: "false"
```

Comment:
This isn't directly related, but used the opportunity to reduce even more duplicates.

Reply:
I think we had to fix the scheduler container for this anyway in our recent rebase: https://github.com/mozilla/redash/commit/ce5e60672f1b49260f5f8b7060dd029b4cac93f2